docs/security/host-certs.md

```diff
@@ -60,20 +60,62 @@ If your institution is not in the list and Let's Encrypt certificates do not mee
 
 1. Generate a Certificate Signing Request (CSR) and private key:
 
+1. Create a custom openssl config for this, using your institution's information for the different fields in the
+   `[ dn ]` section, and the desired hostname for the `CN` field.
+
+        :::console
+        # mycsr.conf
+        [ req ]
+        default_bits = 2048
+        prompt = no
+        encrypt_key = no
+        default_md = sha256
+        distinguished_name = dn
+
+        [ dn ]
+        DC = org
+        DC = incommon
+        C = US
+        postalCode = 53706
+        ST = WI
+        L = Madison
+        street = 1210 West Dayton Street
+        O = University of Wisconsin-Madison
+        OU = OCIS
+        CN = voms.opensciencegrid.org
```

Comment:
Let's only worry about the `Country`, `State` (I believe this needs to be spelled out, e.g. `Wisconsin`), `Locality`, `Organization`, and `CN`. I believe InCommon fills in the rest of the data automatically

Reply:
Let's also suggest a filename for the custom openssl config explicitly instead of having it as a comment in the example file